Gemfile

```diff
@@ -1,22 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
If I add it to the Gemfile, does that also make it a dependency for the entire project?  Or does just the .gemspec determine that when not compiling the project locally?